content/docs/command-reference/gc.md

```diff
@@ -51,8 +51,8 @@ The default remote is cleaned (see `dvc config core.remote`) unless the
 
 ## Options
 
-- `-w`, `--workspace` - keep files and directories _only_ referenced in the
-  current workspace This option is enabled automatically if `--all-tags`,
+- `-w`, `--workspace` - keep _only_ files and directories referenced in the
+  workspace. This behavior is included automatically with `--all-tags`,
   `--all-branches`, or `--all-commits` are used.
```

Comment:
`are used` should be removed now

why do we change this at all? looks very cosmetic - takes time to review

Reply:
Removed "are used".

Changed because "_only_ referenced" means something quite different to "keep _only_". The previous phrasing was incorrect, as they can be references in multiple versions but that doesn't matter nor it it the point. A typo basically

Reply:
I guess the 2nd sentence I didn't need to change. "Included" feels more correct that "enabled" too me but no strong opinion, I can roll that back if you'd like.

Reply:
p.s. I suppose "implied" is even better.

Reply:
👍